troposphere/static/js/components/modals/instance/InstanceLaunchWizardModal.jsx

```diff
@@ -462,7 +462,7 @@ const InstanceLaunchWizardModal = React.createClass({
         });
     },
 
-    filterSizeList(sizes, imageVersion) {
+    filterSizeList(provider, sizes, imageVersion) {
```

Comment:
Oh no, this change should definitely be in master already! I think master is behind z-z. Can you check if that's true before merging?